exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,22 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+let mentor1 = "Daniel";
+let mentor2 = "Irina";
+let mentor3 = "Mimi";
+let mentor4 = "Rob";
+let mentor5 = "Yohannes";
```

Comment:
While the use of `let` is recommended, we can't have the same value declared as above with both `let` and `var`.
Either delete all the mentors' names declarations using `var` and keep the ones using `let`, or delete the `let` and keep the `var`.
It should be fine after that